src/sinks/mod.rs

```diff
@@ -245,10 +244,11 @@ pub enum Sinks {
     #[cfg(feature = "sinks-logdna")]
     Logdna(#[configurable(derived)] logdna::LogdnaConfig),
 
+    /*
```

Comment:
🤔 why the extra comments here?

Reply:
So there's still _two_ sinks -- `loki` and `prometheus_exporter` -- that I can't yet instrument. They need changes to happen in `vector_config` that will be semi-magic-y looking so I wanted to isolate those changes as much as possible for ease of reviewing.

Reply:
Oh whoops yeah - I misread the diff here 😅